src/devrules/core/git_service.py

```
@@ -448,3 +417,101 @@ def get_current_repo_name() -> str:
+    return "main"  # Default fallback
+
+
+def stage_files():
+    """Stage all files for commit."""
+    try:
+        subprocess.run(
+            [
+                "git",
+                "add",
+                "--all",
+            ],
```

Ruff (0.14.13)

Starting a process with a partial executable path

(S607)